packages/extension-countdown/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@jspsych-contrib/extension-countdown",
+  "private": "true",
+  "version": "1.0.0",
```

Comment:
can you make this 0.0.1? The changeset release action will bump the version.

Reply:
Done.